tests/edge-cases/divmod-09.txt

```diff
@@ -0,0 +1,2 @@
+brun '(divmod (q -1) (q -1))'
```

Comment:
We should check all sign combinations. One of those weird int math corners.

https://www.evanjones.ca/integer-division.html